src/Microsoft.DotNet.Darc/src/Darc/Options/GetDependenciesCommandLineOptions.cs

```diff
@@ -11,5 +11,21 @@ internal class GetDependenciesCommandLineOptions : CommandLineOptions
     {
         [Option('n', "name", HelpText = "Name of dependency to query for.")]
         public string Name { get; set; }
+
+        [Option("repo-sha", HelpText = "Get the repo+sha which produced a specific dependency specified by --name or all dependencies in a repository.")]
+        public bool RepoSha { get; set; }
+
+        [Option('l', "local", HelpText = "Required is repo-sha is set and we want to get the repo+sha combinations from local repositories. False by default.")]
```

Comment:
Really not sure what this means, but...

```suggestion
        [Option('l', "local", HelpText = "Required if --repo-sha is set to get the repo+sha combinations from local repositories. False by default.")]
```

or is this more correct?

```suggestion
        [Option('l', "local", HelpText = "Get the repo+sha combinations from local repositories. Requires --repo-sha to be set. False by default.")]
```

Reply:
I like the 2nd better. Thanks for the suggestion!